.github/workflows/project-page-pull-request.yml

```diff
@@ -31,6 +31,10 @@ jobs:
 
         echo "title=$title" >> $GITHUB_OUTPUT
 
+        url=$${{ github.event.issue.html_url }}
+        echo "url [$url]"
+        echo "url=$url" >> $GITHUB_OUTPUT
```

Comment:
Since `url` and `html_url` are two valid attributes, for consistency let's keep `html_url`

![image](https://github.com/NA-MIC/ProjectWeek/assets/219043/126e0dfa-3fad-448b-8ce4-1b688fbfbd23)